includes/class-custom-post-types.php

```diff
@@ -545,13 +545,14 @@ public function append_to_content( $content ) {
 			}
 
 			global $bpfwp_controller;
+
 			if ( $bpfwp_controller->get_theme_support( 'disable_append_to_content' ) ) {
 				return $content;
 			}
 
 			global $post;
-			global $bpfwp_controller;
-			if ( ! is_a( $post, 'WP_POST' ) || $post->post_type !== $bpfwp_controller->cpts->location_cpt_slug ) {
+
+			if ( ! $post instanceof WP_Post || $post->post_type !== $bpfwp_controller->cpts->location_cpt_slug ) {
```

Comment:
oh that sucks, I quite like `is_a()`. Good to know though and I suppose `instanceof` is fine and even more clear.